lib/internal/test_runner/reporter/tap.js

```diff
@@ -113,6 +113,7 @@ function reportDetails(nesting, data = kEmptyObject) {
   let details = `${_indent}  ---\n`;
 
   details += jsToYaml(_indent, 'duration_ms', duration_ms);
+  details += jsToYaml(_indent, 'class', data.class);
```

Comment:
I am not sure about the naming. perhaps `type` instead of `class`

Reply:
what does it actually represent? like the name of the constructor of the object?

Reply:
the type/class of the reported test
currently, a test can either be a suite or a test (there are hooks as well but they are never reported)

Reply:
ah, then definitely "type" or "kind" would make sense